site/app/models/Config.php

```diff
@@ -274,6 +280,8 @@ public function loadMasterConfigs($config_path) {
 
         $this->base_url = $submitty_json['submission_url'];
         $this->submitty_path = $submitty_json['submitty_data_dir'];
+        $this->submitty_install_dir = $submitty_json['submitty_install_dir'];
```

Comment:
need to add this to the list of paths that are validated and then `rtrim` below on lines 332 - 337. Feel free to re-arrange the lines that set paths to be closer to that block to make this connection more clear.

Reply:
thanks for pointing this out!